docs/topics/mtr-rn-new-features.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * docs/release-notes-{LC_PSN}/{LC_PSN}_release_notes-1.0/master.adoc
+
+:_content-type: CONCEPT
+[id="mtr-rn-new-features_{context}"]
+= New features
+
+This section describes the new features of the {ProductName} ({ProductShortName}) 1.0 as compared to Migration Toolkit for Applications (MTA) 5.3.
```

Comment:
This needs to be replaced by MTR

Reply:
@sshveta  I messed it up a few minutes ago. Please take a look now.

Reply:
@sshveta  No, wrong again, sorry.

Reply:
@sshveta  Now, please